aspnetcore/tutorials/web-api-help-pages-using-swagger.md

```diff
@@ -1,11 +1,11 @@
 ---
 title: ASP.NET Web API Help Pages using Swagger
 author: spboyer
-description: Walkthrough of adding Swagger to a Web API Application
-keywords: ASP.NET Core, Swagger
+description: This tutorial provides a walk through of adding Swagger to generate documentation and help pages for a Web API application.
```

Comment:
walkthrough?

Reply:
@innerlee Thank you for the suggestion. I've implemented it.